controllers/event_filters_test.go

```diff
@@ -0,0 +1,56 @@
+package controllers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	dba "github.com/app-sre/dba-operator/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+)
+
+var managedDatabaseUpdatesForPredicateTests = []struct {
```

Comment:
Please move the table directly adjacent to the test that its for.

Reply:
unless you're going to reuse it, just put in in the test func